docs/reference/drawing.rst

```diff
@@ -86,22 +90,22 @@ of nodes of a simple 5-node graph on a small Pegasus lattice.
    >>> import matplotlib.pyplot as plt
    >>> G = dnx.pegasus_graph(2)
    >>> H = dnx.pegasus_graph(2, node_list=[4, 40, 41, 42, 43],
-	             edge_list=[(4, 40), (4, 41), (4, 42), (4, 43)])
+                 edge_list=[(4, 40), (4, 41), (4, 42), (4, 43)])
    >>> # Show graph H on a small Pegasus lattice
    >>> plt.ion()
    >>> # Show graph H on a small Pegasus lattice
    >>> plt.ion()
    >>> dnx.draw_pegasus(G, with_labels=True, crosses=True, node_color="Yellow")
    >>> dnx.draw_pegasus(H, crosses=True, node_color='b', style='dashed',
-	         edge_color='b', width=3)
+             edge_color='b', width=3)
 
 .. figure:: ../_images/pegasus_layout_h_on_g.png
-	:align: center
-	:name: pegasus_layout_h_on_g.png
-	:scale: 60 %
-	:alt: Graph H overlaid on a Pegasus lattice size 2.
+    :align: center
+    :name: pegasus_layout_h_on_g.png
+    :scale: 60 %
+    :alt: Graph H overlaid on a Pegasus lattice size 2.
 
-	Graph H (blue) overlaid on a small Pegasus lattice(yellow nodes and black edges).
+    Graph H (blue) overlaid on a small Pegasus lattice(yellow nodes and black edges).
```

Comment:
Here is a good place to also point out that the layout is rendered in cross format to help users understand what "cross" refers to when used elsewhere (be a good result returned when someone searches "cross")